DESCRIPTION

```diff
@@ -29,6 +29,7 @@ Authors@R: c(
     person("Sunni", "Raleigh", email = "sunnigraleigh@gmail.com", role = "ctb"),
     person("Anna", "Ballou", email = "afballou16@gmail.com", role = "ctb"),
     person("Jane", "Bang", email = "carolbjy@gmail.com", role = "ctb"),
+    person("Kara", "Van Allen", email = "kkv354@gmail.com" role = "ctb"),
```

Comment:
You are missing a space after your email, thus causing the package to not even be able to build.

Reply:
Sorry, you're missing a comma.